### DISCUSSION TITLE
how i can get target device definition from zigbee2mqtt log in zigbe2mqtt?

### DISCUSSION THREAD

**Discussion by kitopopo** [Koenkk/zigbee2mqtt#6225] [General]
Dear @Koenkk ,

I have implemented the v3.0 version of the zigbe2mqtt plugin.
We have lost a functionality in the xiaomi switch (WXKG01LM), now the "many clicks" type of click has disappeared.

I asked mr. stas-demydiuk  if he could implement the click type again "many clicks" in v3 plugin and he tells me that he needs the target device definition from zigbee2mqtt log. 

Where can I get this definition in zigbee2mqtt? I am somewhat lost with this change. 

Thanks in advance

**Comment by Koenkk** [comment]
I don't know what he means with "target device definition".
